arch/arm/soc/microchip_cec/soc_config.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file Board config file
+ */
+
+#include <device.h>
+#include <init.h>
+
+#include <kernel.h>
+
+#include "soc.h"
+
+static int soc_init(struct device *dev)
+{
+	__enable_irq();					/* Enable IRQs as ROM loader does PRIMASK=1 */
+	PCR->CLK_REQ_1 |= 1 << 0;			/* Clock: Interrupt block */
+	PCR->CLK_REQ_1 |= 1 << 8;			/* Clock: Processor */
+#ifdef CONFIG_UART_NS16550_PORT_0
+	PCR->CLK_REQ_2 |= 1 << 1;			/* Clock: UART0 */
+	GPIO_100_137->GPIO_104_PIN_CONTROL = 1 << 12;	/* GPIO104 Alt1 = UART0_TX */
+	GPIO_100_137->GPIO_105_PIN_CONTROL = 1 << 12;	/* GPIO105 Alt1 = UART0_RX */
```

Comment:
We usually do pinmuxing in the board init rather than the SoC init

Reply:
Ah, ok. I'll move the board specifics there. Eventually the pinmuxing will be done from the dts when pinctrl driver works.